cmd/demo/node.go

```diff
@@ -488,3 +514,16 @@ func Setup() {
 	}
 	backend = b
 }
+
+//etherToWei converts amount in "ether" (represented as decimal) to "wei" (represented as integer)
+//It can provide exact results for values in the range of 1e-18 to 1e9
+func etherToWei(ether *big.Float) *big.Int {
```

Comment:
These two conversion methods are better put in the `validator.go` file since we also have one other conversion method there.

Reply:
Addressed in this commit - e10c196.